packages/patterns-react/src/patterns/SimpleLongForm/SimpleLink.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { ArrowRight16 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Simple Link Component
+ *
+ * @param {*} { link: { href, text, target } }
+ * @returns {*} SimpleLink component
+ */
+/**
+ * Simple Link Component
+ *
+ * @param {object} props props object
+ * @param {string} props.href simple link link
+ * @param {string} props.target simple link text
+ * @returns {*} SimpleLink component
+ */
+const SimpleLink = ({ link: { href, target } }) => {
+  return (
+    <a
+      data-autoid={`${stablePrefix}--simplelink`}
+      className={`${prefix}--simplelink`}
+      href={href}
+      target={target === 'blank' ? '_blank' : '_self'}>
+      <span className={`${prefix}--simplelink__text`}>Learn more</span>
```

Comment:
Can this be set via prop? We shouldn't have english burned in here.

Reply:
replaced learn more english text with props by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/8df6d576041863cd9f0b34f4da3407f78fde4f76)